dct_backend/reviews.py

```diff
@@ -0,0 +1,28 @@
+from fastapi import FastAPI, Response, status
+from dotenv import load_dotenv
+import os
+from database import Database
+
+load_dotenv()
+db = Database(os.getenv("DB_USERNAME"), os.getenv("DB_PASSWORD"))
+db.connect()
+food_db = db.client["food"]
+review_collection = food_db["reviews"]
+resp = Response
+
+class Reviews():
+    def save_review_in_db(review_rating: dict):
+        if review_collection.count_documents({"_id": review_rating['_id']}, limit = 1) > 0: # each user can add one review per food item
```

Comment:
So the `_id` in `review_rating` refers to the row entry `_id`, NOT the user `_id` you will need a separate entry: `uuid` that links the food review to the user and check that.